### DISCUSSION TITLE
Be able to edit code from the app

### DISCUSSION THREAD

**Discussion by BlakeAyrl** [community/community#5443] [Mobile] [answered]
Looking to be able to edit code from the app to program. Its 2021 this shouldn't be difficult specially with competitors doing this already.

**Comment by aconsuegra** [comment]
Thanks for your feedback! We are tracking this request internally and are looking into how we can bring this to the app.

**Reply by FollowNaim** [reply]
You have been saying the same thing for six or seven months.
Do you really do it or not?
if yes then How long will it take?

**Reply by verlok** [reply]
Thanks. I'm looking forward to being able to edit code from the app too. 

In the meantime, I'll keep doing it from the mobile website, even though it's very uncomfortable.

**Comment by spaltrowitz** [comment, marked as answer]
thanks for your suggestion @BlakeAyrl! this has now shipped https://github.blog/changelog/2022-11-15-github-for-ios-edit-files-from-browse-code
